frontend/src/metabase/common/components/CollectionPicker/components/CollectionItemList.tsx

```diff
@@ -1,23 +1,29 @@
-import { useSearchListQuery } from "metabase/common/hooks";
+import { skipToken, useListCollectionItemsQuery } from "metabase/api";
 
 import { ItemList } from "../../EntityPicker";
 import type { CollectionItemListProps, CollectionPickerItem } from "../types";
 
-export const SearchItemList = ({
+export const CollectionItemList = ({
   query,
   onClick,
   selectedItem,
   isFolder,
   isCurrentLevel,
   shouldDisableItem,
 }: CollectionItemListProps) => {
-  const { data, error, isLoading } = useSearchListQuery<CollectionPickerItem>({
-    query,
-  });
+  const {
+    data: collectionItems,
+    error,
+    isLoading,
+  } = useListCollectionItemsQuery<{
+    data: { data: CollectionPickerItem[] };
+    error: any;
+    isLoading: boolean;
```

Comment:
why are you having to do this? just because of the skip token? and why does the data key have a data key? doesn't seem like this is reflected in the `ListCollectionsResponse` type.

Reply:
this is needed to get a `CollectionPickerItem` instead of a `CollectionItem`